ArcGISRuntimeSDKQt_CppSamples/Analysis/Geotriggers/Geotriggers.cpp

```diff
@@ -229,22 +229,20 @@ void Geotriggers::getFeatureInformation(const QString& sectionName)
 
   emit displayInfoChanged();
 
-  AttachmentListModel* featureAttachments = feature->attachments();
-
-  // Fetch attachments will automatically trigger upon instantiation of the AttachmentListModel
-  connect(featureAttachments, &AttachmentListModel::fetchAttachmentsCompleted, this, [this, sectionName](const QUuid&, const QList<Attachment*>& attachments)
-  {
-    if (attachments.isEmpty())
-      return;
-
-    // Get the first (and only) attachment for the feature
-    Attachment* sectionImageAttachment = attachments.first();
-
-    sectionImageAttachment->fetchDataAsync().then(this, [this, sectionImageAttachment, sectionName](const QByteArray&)
-    {
-      m_featureAttachmentImageUrls[sectionName] = sectionImageAttachment->attachmentUrl();
-      m_currentFeatureImageUrl = m_featureAttachmentImageUrls[sectionName];
-      emit displayInfoChanged();
-    });
-  });
+  feature->attachments(false, false)->fetchAttachmentsAsync().then(
```

Comment:
This could be confusing. This is a cached object, so I recommend storing AttachmentListModel as a class member so you don't need to call `feature->attachments(false, false)` multiple times. 

Having repeated calls to `feature->attachments(false, false)` gives the impression that users can change the arg values and change the behavior every time, which is not how it works.

Reply:
Stored AttachmentListModel as a class member.

Reply:
Reverted back to `feature->attachments(false, false)` as the latest comment.